views/index.hbs

```diff
@@ -1,2 +1,11 @@
 <h1>{{title}}</h1>
-<p>Welcome to {{title}}</p>
\ No newline at end of file
+<p>Welcome to {{title}}</p>
+
+<a href="/celebrities/create">Create new Celebrity</a>
+
+<a href="/celebrities">View all Celebrities</a>
+
+<a href="/movies/create">Create a new Movie</a>
+
+<a href="/movies">View all Movies</a>
```

Comment:
I would put the `<a>` in the `layout.hbs` so that would work as a navbar for the whole website